docs/api.rst

```diff
@@ -71,6 +71,12 @@ like          :code:`LIKE`              LIKE operator (to avoid `URL encoding <h
 ilike         :code:`ILIKE`             ILIKE operator (to avoid `URL encoding <https://en.wikipedia.org/wiki/Percent-encoding>`_ you can use ``*`` as an alias of the percent sign ``%`` for the pattern)
 match         :code:`~`                 ~ operator, see :ref:`pattern_matching`
 imatch        :code:`~*`                ~* operator, see :ref:`pattern_matching`
+all           :code:`ALL`               comparison matches all the values in the list,
+                                        used as a modifier of :code:`eq,like,ilike,gt,gte,lt,lte,match,imatch`,
+                                        e.g. :code:`?name=like(all).{J*,*e}`
+any           :code:`ANY`               comparison matches any value in the list,
+                                        used as a modifier of :code:`eq,like,ilike,gt,gte,lt,lte,match,imatch`,
+                                        e.g. :code:`?name=like(any).{J*,*e}`
```

Comment:
This should be at the end of the table, next to the `or` and `and` operators.

It could also have a mention in the "Logical Operators" section. Some advantages:

- `any` values don't need double quoting unlike `in` values
- They really make more sense by using the `like/ilike` operators. `all` variant can be used as an example in combination with `like`.
- They're more of a shortcut. Check https://github.com/PostgREST/postgrest/issues/1569